simd/keccakf1600/f1600x4_test.go

```diff
@@ -1,14 +1,14 @@
-package f1600x4
+package keccakf1600
 
 import (
 	"testing"
 )
 
 func TestF1600x4(t *testing.T) {
-	if !Available {
+	if !AvailableX4 {
```

Comment:
should test/bench the generic implementation too.

Reply:
I added a test for the generic implementation.